gtda/images/filtrations.py

```diff
@@ -1036,3 +1038,228 @@ def plot(Xt, sample=0, colorscale='greys', origin='upper',
             title=f"Signed-distance filtration of image {sample}",
             plotly_params=plotly_params
             )
+
+
+@adapt_fit_transform_docs
+class DensityFiltration(BaseEstimator, TransformerMixin, PlotterMixin):
+    """Filtrations of 2D/3D binary images based on the number of activated
+    neighboring pixels.
+
+    The density filtration assigns to each pixel of a binary image a greyscale
+    value equal to the sum of the distance between this pixel and all activated
```

Comment:
Do you really mean the sum of the distances? Or do you mean the number in a ball centered around the pixel?

Reply:
There is an array that represent the distances to the center of a ball. This mask is simply convoluted over all pixels of the image.

Reply:
@gtauzin so that's a yes? I ask because it seems to me to be a weird measure of density. In fact I'd have expected something like the sum of inverse distances (we are more confident about immediate neighbours as a measure of density, as opposed to further away ones).

Reply:
To flesh out my point (if there is any): compare two points A and B. A is surrounded by nothing except for a dense but thin annulus of activated pixels just before distance R, while B is surrounded by a cloud of the same number of activated pixels within distance R, but all closer to B. We would then say that there is higher density around A compared to B, which seems to go against intuition.

Reply:
That's my bad. It should just count the number of activated pixels within a ball defined by radius and metric. The doc have been updated.

Reply:
Let me know if the new doscstrings work for you!